### ISSUE TITLE
Add stardist to Bioconda

### ISSUE THREAD

**Issue by migueLib** [stardist/stardist#259]
TL;DR
We would want to contribute in anyway possible to integrate stardist into [bioconda](https://bioconda.github.io/) in the near future! 

Dear stardist team,

First of all thanks for the effort you have been putting into this tool!  It's been a pleasure to use.

I'm a PhD candidate at University Heidelberg at the [SchapiroLab](http://schapirolab.com/) - [github](https://github.com/SchapiroLabor) , together with @kbestak, @VictorDidier and @josenimo are interested add stardist as a [Bioconda](https://bioconda.github.io/) package.  This, with the purpose of fully integrate stardist as part of [MCMICRO](https://mcmicro.org), an end-to-end processing pipeline that transforms multi-channel whole-slide images into single-cell data. 

MCMICRO uses [Nextflow](https://www.nextflow.io) and [nf-core](https://nf-co.re) guidelines to integrate the different steps needed. nf-core guidelines suggest Conda, Docker and Singularity containers for the creation of a module. We are aware of the existence of the existence of stardist as a [conda-forge](https://anaconda.org/conda-forge/stardist) package, we  are also aware of the existence of the Dockerfile but we haven't found an associated docke container (we have found one docker container but seems to be outdated and we are not sure if you  support it https://quay.io/repository/goeckslab/stardist?tab=tags).  

Bioconda provides a build system that turns each recipe into a conda package and a Docker container, we are wondering if you had consider adding stardist here and if not how interested would you be to do so. We are more than happy to contribute in any meaningful way to make this happen. 

Excited to hear from you! 
Bests!

**Comment by maweigert** [comment]
Hi @migueLib ,

Thanks for this message and yes, we'd be very happy to have it added to bioconda :)
Maybe you could detail a bit the process that would be necessary to do so - pinging @uschmidt83 too.

Cheers,
Martin

**Comment by FloWuenne** [comment]
@maweigert We are having some issues related to the nvidia driver specification in the docker file. Is there a way to build/ run stardist without the GPU requirement? If GPU is critically required, do you know if there is a way to specify this so it works across platforms and in singularity. Specifying GPU at build wouldn't really help much for a hosted Docker container and singularity builds.

**Comment by uschmidt83** [comment]
@FloWuenne I replied in https://github.com/BioContainers/containers/pull/551, let's continue the discussion (regarding the specifics of the Dockerfile) there.
